change/@fluentui-react-carousel-973e8fc6-d327-490f-a135-a9100a53033c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=33000&commitId=b5ff12ec59e41ccbc0db12e68a79264ce658c945&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|10| Changed|
|Avatar Converged.badgeMask.chromium.png|61| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.overlay drawer full.chromium.png|1138| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>